mlir/test/lib/IR/CMakeLists.txt

```diff
@@ -27,7 +27,11 @@ add_mlir_library(MLIRTestIR
   TestVisitorsGeneric.cpp
 
   EXCLUDE_FROM_LIBMLIR
+
+  DEPENDS
```

Comment:
Don't you need `LINK_LIBS` here too?

Reply:
I do, in fact - ad152f4bcfe465b57562fa003b93f44e1a3b2287.

Will pick this too.